packages/mdc-dialog/README.md

```diff
@@ -400,6 +405,8 @@ Method Signature | Description
 `setScrimClickAction(action: string)` | Sets the action reflected when the scrim is clicked. Setting to `''` disables closing the dialog via scrim click.
 `getAutoStackButtons() => boolean` | Returns whether stacked/unstacked action button layout is automatically handled during layout logic.
 `setAutoStackButtons(autoStack: boolean) => void` | Sets whether stacked/unstacked action button layout is automatically handled during layout logic.
+`getInitialFocusEl() => HTMLElement|null` | Gets the element to focus on after dialog has finished opening.
+`setInitialFocusEl(el: HTMLElement|null)` | Sets the element to focus on after dialog has finished opening. Passed as an argument to `Adapter#trapFocus`.
```

Comment:
Let's add a line about this in `Handling Focus Trapping` section.

Reply:
Not applicable since we removed this method.